dfc/proxy.go

```diff
@@ -1613,6 +1626,33 @@ func (p *proxyrunner) tokenhdlr(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// handler for dfc being used as a HTTP proxy
+func (p *proxyrunner) proxyhdlr(w http.ResponseWriter, r *http.Request) {
```

Comment:
this name will be very confusing for people read the code for the first time, including both "reverse proxy" probably better